lib/ReactViews/Story/StoryPanel.jsx

```diff
@@ -159,14 +159,7 @@ const StoryPanel = observer(
     },
 
     onCenterScene(story) {
-      if (story.shareData) {
-        this.props.terria
-          .updateFromStartData(
-            story.shareData,
-            `Story data: \`${story.title ?? story.id}\``
-          )
-          .raiseError(this.props.terria);
-      }
+      activateStory(story, this.props.terria);
     },
```

Comment:
Can't think of a reason why we shouldn't be using `activateStory` instead of `updateFromStartData`. Calling `updateFromStartData` has a few problems:
1. Every time you "center the story" it adds a duplicate story share data in `Terria.initSources`
2. There is no way to pass `replaceStratum` so that we can get a clean state when resetting the scene.

When looking at the history of `onCenterScene` in `v7` branch, it looks like to me that both `activateStory` and `onCenterScene` used to call `updateFromStartData` then changes were made to activateStory but not to onCenterScene, which could be an error.